### PR TITLE
Aggregated updates for UMAPI documentation

### PR DESCRIPTION
**Pull request by dscoleman-w**
Reopened PR with redacted files removed.

### CODE REVIEW

**Comment by dscoleman-w** [comment]
Previous PR comments available in attachement.

[Aggregated updates for UMAPI documentation by dscoleman-w · Pull Request #6 · adobe-apiplatform_umapi-documentation.pdf](https://github.com/adobe-apiplatform/umapi-documentation/files/1296725/Aggregated.updates.for.UMAPI.documentation.by.dscoleman-w.Pull.Request.6.adobe-apiplatform_umapi-documentation.pdf)

File: docs/_config.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ description: > # this means to ignore newlines until "baseurl:"
   The User Management API allows clients to manage their users
   and product entitlements.
   ***This is a draft development area.***
-baseurl: "/umapi-documentation" # the subpath of your site, e.g. /blog
+baseurl: "" # the subpath of your site, e.g. /blog
```
Comment (phil-levy) [inline docs/_config.yml]: Still needs fixing

File: docs/_includes/apiRef/badRequest.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+### <a name="{{ include.anchor }}" class="api-ref-subtitle">400 Bad Request</a>
+Some parameters of the request were not understood by the server or the [Service Account Integration](https://adobe-apiplatform.github.io/umapi-documentation/en/getstarted.html) certificate has expired.
+
+{% if include.object == "user-group" %}
+Possible cause:
+- Group name already exists
+
+```
+> POST /v2/usermanagement/users/092DE2D65617B9967F000101@AdobeOrg/user-groups HTTP/1.1
```
Comment (phil-levy) [inline docs/_includes/apiRef/badRequest.md]: This is still pretty leaky about user-group operations

File: docs/_plugins/auto_nav_generator.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Navigation Generator is a Jekyll plugin that extracts H2 headings from each page so that it can be used as part of the generation of the navigation menu.
```
Comment (phil-levy) [inline docs/_plugins/auto_nav_generator.rb]: This will need to be removed.

File: docs/_includes/apiRef/apiKeyDescription.md [hunk 4]
```diff
@@ -0,0 +1 @@
+The API key specified in the [Adobe IO Console](https://console.adobe.io/) <button id="apiKeyDescriptionButton" onclick="showApiKeyDescription()">(...)</button><span id="apiKeyDescription">for the UMAPI integration used to authorize this session.  See the [Getting Started](https://adobe-apiplatform.github.io/umapi-documentation/en/getstarted.html) documentation for details.  The header name is `X-Api-Key` and this parameter is its value.</span>
```
Comment (phil-levy) [inline docs/_includes/apiRef/apiKeyDescription.md]: Button things need removal.

File: docs/en/api/.creatUserGroup.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+```json
+{
+    "groupId": long,
+    "name": "string",
+    "type": "string"
+  }
+```
+
+{% include apiRef/badRequest.md object="user-group" anchor="400createUserGroup" %}
```
Comment (phil-levy) [inline docs/en/api/.creatUserGroup.md]: Need to replace with include_relative
Same for all {%includes

File: docs/en/index.md [hunk 6]
```diff
@@ -9,14 +9,12 @@ lang: en
 
 Welcome to the documentation center for User Management APIs from Adobe.
 
-**This is a draft development area.  Production documentation is [here](https://www.adobe.io/apis/cloudplatform/usermanagement/docs/gettingstarted.html)**
+**This is a draft development area.**  Production documentation is [here](https://www.adobe.io/apis/cloudplatform/usermanagement/docs/gettingstarted.html).
 
-[Getting Started with User Management](getstarted.md)
+[Getting Started with User Management](getstarted.html)
```
Comment (phil-levy) [inline docs/en/index.md]: Still need to resolve these (and all .html references)

File: docs/_plugins/auto_nav_generator.rb (hunk 3 above)
Comment (dscoleman-w) [reply docs/_plugins/auto_nav_generator.rb]: Lea is working on this. We will remove this later in the day.

File: docs/_includes/apiRef/apiKeyDescription.md (hunk 4 above)
Comment (dscoleman-w) [reply docs/_includes/apiRef/apiKeyDescription.md]: We will address this point offline.

File: docs/en/index.md (hunk 6 above)
Comment (dscoleman-w) [reply docs/en/index.md]: This will be taken up in another PR.

**Pull request merged**